src/interfaces.ts

```diff
@@ -0,0 +1,61 @@
+export interface Token {
+  kind: TokenKind;
+  value: string;
+  position: Position;
+}
+
+export interface Position {
+  line: number;
+  column: number;
+  offset: number;
+}
+
+export enum TokenKind {
+  Assign = "Assign",
```

Comment:
@tailhook I'm curious if there's a way to reflect these automatically...

Reply:
Yes, kinda: https://github.com/edgedb/edgedb-js-lexer/pull/4
This generates:
```typescript
export enum TokenKind {
  Assign,
  SubAssign,
  AddAssign,
  // ...
```
So without string values. And we still can't export list of our own classes from wasm. So it must be casted with `TokenKind[token.kind]`. But it's better than supporting the separate list anyway.

Reply:
@jaclarke James, I've merged Paul's #4 PR. Please take a look if you need anything else. IMO, a non-string enum should be OK.

Reply:
@tailhook Do you know if there's a way to export `TokenKind` into it's own file, separate from the main `edgeql_wasm_bg.js` file? There may not be, I can only find an option to set the name of the export in the wasm_bindgen docs, in which case I can just extract it in some post build step.

Reply:
I can't find any way to do that.